### PR TITLE
Jorgen/82 api endpoint for doc

### PR DESCRIPTION
**Pull request by jbtfk**
Added endpoint for fetching documentation for sink

### CODE REVIEW

File: app/Http/Controllers/SinkApiController.php [hunk 1]
```diff
@@ -77,6 +78,17 @@ public function scanLocalFiles(Sink $sink): Response
         ]);
     }
 
+    /**
+     * asd
+     */
+    public function getDoc(Sink $sink): Response
+    {
+        $this->authorize('view', $sink);
```
Comment (tfk-kaare) [inline app/Http/Controllers/SinkApiController.php:86]: Yess!

File: app/Http/Controllers/SinkApiController.php [hunk 2]
```diff
@@ -77,6 +78,17 @@ public function scanLocalFiles(Sink $sink): Response
...
+        $res = Ragnarok::getSinkHandler($sink->id)->getSinkDocumentation();
+        $scode = $res === null ? 204 : 200;
+        return response($res, $scode)->header('Content-Type', 'text/html');
```
Comment (tfk-kaare) [inline app/Http/Controllers/SinkApiController.php:89]: `$res` is often used as request result stuff, but what you get here is documentation, so e.g.
```php
        $doc = Ragnarok::getSinkHandler($sink->id)->getSinkDocumentation();
        return response($doc, $doc ? 204 : 200)->header('Content-Type', 'text/html');
```

**Review by tfk-kaare — COMMENTED** [review]
Good, clean code. Minor nit regarding var name …

File: app/Http/Controllers/SinkApiController.php (hunk 1 above)
Comment (jbtfk) [reply app/Http/Controllers/SinkApiController.php:89]: Yess? OK

Comment (jbtfk) [reply app/Http/Controllers/SinkApiController.php:89]: Added proper documentation block to function

File: app/Http/Controllers/SinkApiController.php (hunk 2 above)
Comment (jbtfk) [reply app/Http/Controllers/SinkApiController.php]: Fixed

**Review by tfk-kaare — APPROVED** [review]
Hunkydory

**Pull request merged**